rawkit/raw.py

```diff
@@ -7,6 +7,7 @@
 import random
 import string
 import tempfile
+import numpy as np
```

Comment:
Yup, that should be fine. You won't be able to use the numpy methods to do it though; just regular iteration will work fine.